frontend/src/views/Login/Login.jsx

```diff
@@ -53,8 +53,8 @@ function Login() {
   const onSubmit = ({ username, password }) => {
     executeLogin({ data: { username, password } })
       .then(({ data }) => {
-        document.cookie = JSON.stringify(data);
-        history.push('/movies');
+        document.cookie = JSON.stringify(data).substring(1, data.length);
```

Comment:
```suggestion
        document.cookie = JSON.stringify(data).substring(1);
```
when your end is the length size you could just not write it

Reply:
ok thanks